code-compass.el

```diff
@@ -613,216 +623,223 @@ code can infer it automatically."
+                    (code-compass--second it)
+                  (s-concat repository "/" (code-compass--second it))))
+               (cdr (cdr it)))
+              (s-join "," it))
+         (cdr it)))))
+
+(defun code-compass--get-coupling-alist-sync (repository)
   "Get list of coupled files in REPOSITORY async."
-  (c/in-temp-directory
+  (code-compass--in-temp-directory
    repository
    (--> repository
-     (c/produce-git-report it nil)
-     c/produce-code-maat-coupling-report
-     (c/get-analysis-as-string-from-csv it "coupling")
-     (c/add-filename-to-analysis-columns repository it)
+     (code-compass-produce-git-report it nil)
+     code-compass--produce-code-maat-coupling-report
+     (code-compass--get-analysis-as-string-from-csv it "coupling")
+     (code-compass--add-filename-to-analysis-columns repository it)
      (--map (s-split "," it) (cdr it)))))
 
-(defun c/get-coupling-alist (repository fun)
+(defun code-compass--get-coupling-alist (repository fun)
   "FUN takes a list of coupled files in REPOSITORY."
   (async-start
    `(lambda ()
       (setq load-path ',load-path)
-      (load-file ,(symbol-file 'c/get-coupling-alist))
-      (c/get-coupling-alist-sync ,repository))
+      (load-file ,(symbol-file 'code-compass--get-coupling-alist))
+      (code-compass--get-coupling-alist-sync ,repository))
    fun))
 
 
-(defvar c/coupling-project-map
+(defvar code-compass-coupling-project-map
   (make-hash-table :test 'equal)
   "Hash table to contain coupling files list.")
 
-(defun c/get-coupled-files-alist (repository fun)
+(defun code-compass--get-coupled-files-alist (repository fun)
   "Run FUN on the coupled files for REPOSITORY."
-  (let* ((key (funcall c/calculate-coupling-project-key-fn repository))
-         (c/files (gethash key c/coupling-project-map)))
-    (if c/files
-        (funcall fun c/files)
+  (let* ((key (funcall code-compass-calculate-coupling-project-key-fn repository))
+         (code-compass-files (gethash key code-compass-coupling-project-map)))
+    (if code-compass-files
+        (funcall fun code-compass-files)
       (progn
         (message "Building coupling cache asynchronously...")
-        (c/get-coupling-alist
+        (code-compass--get-coupling-alist
          repository
          `(lambda (result-files)
             (message "Coupling Cache Built")
-            (puthash ,key result-files c/coupling-project-map)
+            (puthash ,key result-files code-compass-coupling-project-map)
             (funcall ,fun result-files)))))))
 
-(defun c/clear-coupling-project-map ()
-  "Clear `c/coupling-project-hash'."
+(defun code-compass-clear-coupling-project-map ()
+  "Clear `code-compass-coupling-project-map'."
   (interactive)
-  (clrhash c/coupling-project-map))
+  (clrhash code-compass-coupling-project-map))
+(defalias 'c/clear-coupling-project-map 'code-compass-clear-coupling-project-map)
 
-(defun c/get-coupled-files-alist-hook-fn ()
```

Comment:
could you leave this as a public API and alias it as well? `code-compass-get-coupled-files-alist-hook-fn`

Reply:
done